x-pack/test/apm_api_integration/tests/alerts/anomaly_alert.spec.ts

```diff
@@ -23,23 +23,24 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   const logger = getService('log');
 
   const synthtraceEsClient = getService('synthtraceEsClient');
-  // FLAKY https://github.com/elastic/kibana/issues/160298
-  registry.when.skip(
+  registry.when(
     'fetching service anomalies with a trial license',
     { config: 'trial', archives: [] },
     () => {
-      const start = '2021-01-01T00:00:00.000Z';
-      const end = '2021-01-08T00:15:00.000Z';
+      const start = Date.now() - 1000 * 60 * 60 * 24 * 2; // day ago
+      const end = Date.now();
```

Comment:
nit: `moment` with `subtract()` and `add()` is easier to read.

Reply:
Changed here https://github.com/elastic/kibana/pull/169255/commits/bf3d743e42217cd02b9f894536a04ccea6d172e9